### DISCUSSION TITLE
Could not connect to server

### DISCUSSION THREAD

**Discussion by sTiKyt** [firefly-iii/firefly-iii#6624] [Firefly III] [answered]
I'm trying to setup it with postgres container on Oracle Linux server

But i get this:
![image](https://user-images.githubusercontent.com/35602029/200912363-d96a00af-698d-483b-a9d8-2500436442f2.png)

![image](https://user-images.githubusercontent.com/35602029/200912117-f5fc94da-d536-466d-9cb1-d52d0da96d13.png)

Containers seem to work fine:
![image](https://user-images.githubusercontent.com/35602029/200912501-3b446d25-dd1d-4e1a-94f4-cc330bfd788a.png)
![image](https://user-images.githubusercontent.com/35602029/200912751-bb11ae7c-3d8e-483b-8929-d2c3ecb8c292.png)

**Comment by mtb-xt** [comment, marked as answer]
@sTiKyt 127.0.0.1 means `localhost` - the same machine as firefly 3. You have postgres in another container - so you probably need to use it's container name (and make sure those two containers are linked - they can reach each other)

**Reply by sTiKyt** [reply]
Oh, makes sense, I'm not used to docker so wasn't sure, anyway I've setup it using Docker Compose and it works great, thanks!
